R/mig_resid.R

```diff
@@ -579,18 +584,20 @@ mig_resid_stock <- function(pop_m_mat,
     is.numeric(ages),
     is.numeric(ages_fertility)
   )
-  # Check in dimensions are ok
-  if(ncol(asfr_mat) == ncol(pop_f_mat)-1){
+  # Check in dimensions are ok - still working on this
```

Comment:
hi @tbernardesfaria I was thinking these steps (all the dim checking, etc) would be modularized. That means that all these checks, which take up many lines, would live in their own new function, and that this function would be called near the head of `mig_resid_stock()`, `mig_resid_cohort()`, and `mig_resid_time()`. Otherwise, the code would need to be repeated in 3 places.